app/front_page/content.html

```diff
@@ -97,44 +97,49 @@
 </ui-minimize-panel>
 
 <ui-minimize-panel title="File list" open='false'>
-  <div class='panel panel-default'>
+  <div class='panel panel-default' ng-controller="FileListController as vm">
     <div class='panel-heading'>
       File List
-    </div>
-    <div class='panel-body' ng-controller="FileListController as vm">
-      <div ng-if="vm.selected_files.selected.length > 0">
-        <p><b><ng-pluralize count="(vm.selected_files.selected | filter: {'type': 'file'} | facet).length" when="{'1': '1 object', 'other': '{} objects'}"></ng-pluralize></b> in list, <ng-pluralize count="vm.selected.length" when="{'1': '1 object', 'other': '{} objects'}"></ng-pluralize> selected</p>
 
-        <fieldset>
-          <label for='start-date-facet'>Date range start</label>
-          <input type="text"
-                 ng-model="vm.date_start"
-                 class='form-control'
-                 id='start-date-facet'
-                 ng-change="vm.set_date_filter(vm.date_start, vm.date_end)"
-                 ng-pattern="vm.date_regex"/>
-          <label for='end-date-facet'>Date range end</label>
-          <input type="text"
-                 ng-model="vm.date_end"
-                 class='form-control'
-                 id='end-date-facet'
-                 ng-change="vm.set_date_filter(vm.date_start, vm.date_end)"
-                 ng-pattern="vm.date_regex"/>
-        </fieldset>
-        <span ng-if="vm.date_facet">Dates: {{ vm.date_facet }} <span ng-click="vm.reset_dates();"><i class="fa fa-times"></i></span></span>
+      <span class="counts" ng-if="vm.selected_files.selected.length > 0"><b><ng-pluralize count="(vm.selected_files.selected | filter: {'type': 'file'} | facet).length" when="{'1': '1 object', 'other': '{} objects'}"></ng-pluralize></b> in list, <ng-pluralize count="vm.selected.length" when="{'1': '1 object', 'other': '{} objects'}"></ng-pluralize> selected</span>
+
+      <span class="counts" ng-if="vm.selected_files.selected.length === 0">File list is empty.</span>
+
+      <!--
+      <fieldset>
+        <label for='start-date-facet'>Date range start</label>
+        <input type="text"
+               ng-model="vm.date_start"
+               class='form-control'
+               id='start-date-facet'
+               ng-change="vm.set_date_filter(vm.date_start, vm.date_end)"
+               ng-pattern="vm.date_regex"/>
+        <label for='end-date-facet'>Date range end</label>
+        <input type="text"
+               ng-model="vm.date_end"
+               class='form-control'
+               id='end-date-facet'
+               ng-change="vm.set_date_filter(vm.date_start, vm.date_end)"
+               ng-pattern="vm.date_regex"/>
+      </fieldset>
+      <span ng-if="vm.date_facet">Dates: {{ vm.date_facet }} <span ng-click="vm.reset_dates();"><i class="fa fa-times"></i></span></span>
+      -->
```

Comment:
Is it necessary to commit commented out code? I think there is a valid use case when you want to document an example but this doesn't seem to be the case.